content/docs/getting-started/app-maintenance.md

```diff
@@ -22,3 +22,21 @@ Whenever you deploy an app update with `cf push`, your app will use the latest b
 ## Avoiding downtime during buildpack or app updates
 **Note that your app may be unavailable while a `cf restage` or `cf push` is in progress.** Avoid this by [setting up zero-downtime deployments for your app](https://cloud.gov/docs/apps/production-ready/#zero-downtime-deploy), so that every new version of your app picks up the latest buildpack without impacting your app's availability.
 
+## Operating system patching
+
+The cloud.gov platform team is responsible for maintenance and patching of the operating system (Ubuntu) and other platform components. Your applications benefit from these updates automatically.
```

Comment:
I wouldn't explicitly name Ubuntu here. CF supports other distros and we'll definitely have Windows cells in future. The vast majority of applications installed with a buildpack don't even care what the distro is.